src/main.js

```diff
@@ -17,3 +23,25 @@ document.addListenerToButtons = addListenerToButtons(
     'what starship from Star Wars is showed in the picture.',
   ]
 );
+
+const intervalId = setInterval(() => {
```

Comment:
As the intro is visible only during the fixed time, a solution to the thing you asked could be closing the intro only after this fixed time passed by using `setTimeout`. I think you don't have to find the moment the last line of the intro text disappears as the text stays the same every time the intro is displayed.